services/core/MasterDriverAgent/master_driver/interfaces/modbus_tk/tests/modbus_server.py

```diff
@@ -1,5 +1,5 @@
 from master_driver.interfaces.modbus_tk.server import Server
-from master_driver.interfaces.modbus_tk.define import *
+from master_driver.interfaces.modbus_tk import helpers
```

Comment:
Thank you. This makes your code easier to understand.

Reply:
I fixed it.